app/assets/stylesheets/govuk_publishing_components/components/_single-page-notification-button.scss

```diff
@@ -0,0 +1,32 @@
+.gem-c-single-page-notification-button__submit {
+  position: relative;
+  padding: govuk-spacing(2);
+  text-decoration: none;
+  margin: govuk-spacing(0);
+  box-shadow: inset 0 0 0 1px transparent;
```

Comment:
On the line you made the comment on: I don't think it's needed actually, so I have removed it.

On the focus state: the `box-shadow` property is used to apply the thicker line on the bottom of the button. The reason you'd want to use `box-shadow` to do so instead of say, `border-bottom`, is to avoid layout shift.